### PR TITLE
Microsite redesign

### PR DESCRIPTION
**Pull request by TiannaW24**
Create the new microsite structure and content, delete the old stuff.

### CODE REVIEW

File: banner.html [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  var current_page = document.getElementById('banner').getAttribute('data-current-page');
+  document.getElementById(current_page).classList.add('usa-current')
+  if (current_page == 'home_menu') {
+    document.getElementById('last_hr').setAttribute('hidden', 'true');
+  }
+</script>
+<a class="usa-skipnav" href="#main-content">Skip to main content</a>
+<div class="usa-banner">
+  <div class="usa-accordion">
+    <header class="usa-banner__header">
```
Comment (aaronFranssell) [inline banner.html]: Is this header different than previous headers?

File: about.html [hunk 2]
```diff
@@ -1,194 +1,118 @@
...
-<meta name="author" content="U.S. Department of Transportation">
-<!-- Disable tap highlight on IE -->
-<meta name="msapplication-tap-highlight" content="no">
-<title>About SDC | Secure Data Commons</title>
-<!-- USWDS Styles -->
-<link rel="stylesheet" href="assets/css/uswds.min.css">
-<!-- SDC Styles -->
-<link rel="stylesheet" href="css/styles.css">
+  <!-- USWDS Styles -->
+  <link rel="stylesheet" href="assets/css/uswds.min.css">
```
Comment (aaronFranssell) [inline about.html]: Can you move these stylesheet declarations into the shared header as well?

File: banner.html [hunk 3]
```diff
@@ -0,0 +1,80 @@
+<script>
+  var current_page = document.getElementById('banner').getAttribute('data-current-page');
```
Comment (aaronFranssell) [inline banner.html]: It appears that `load` will also execute any scripts contained in the page... Can you rewrite this in Jquery? Something like:
```
$( document ).ready(function() {
  var current_page = $('#banner').data('current-page');
  $('#' + current_page).addClass('usa-current');
  if (current_page == 'home_menu') {
    $('#last_hr').hide();
  }
});
```

File: about.html (hunk 2 above)
Comment (TiannaW24) [reply about.html]: If I move them there, there is a visible "blink" of the styles loading on the page. The user will see the page without styles for a second and then the styles will load.

File: banner.html (hunk 3 above)
Comment (TiannaW24) [reply banner.html]: done

File: sdc_lifecycle.html [hunk 4]
```diff
@@ -0,0 +1,192 @@
...
+              </li>
+            </ul>
+          </nav>
+        </div>
+        <div class="tablet:grid-col-10">
+          <h1 class="margin-top-0 margin-bottom-1" style="font-weight: normal;">SDC Lifecycle</h1>
+          <p class="text-normal">Each project in the SDC goes through a standard lifecycle of activities.</p>
+        </div>
+        <div class="lifecycleInfoCard tablet:grid-offset-2 tablet:grid-col-8 margin-bottom-neg-2" >
+          <div style="border: solid #9DDFEB 8px;" >
```
Comment (aaronFranssell) [inline sdc_lifecycle.html]: Can you remove all inline styles? This `9DDFEB` color in particular is in a couple places...

**Pull request merged**